extras/Dockerfile.simple-quota-crawler

```diff
@@ -0,0 +1,8 @@
+FROM alpine
+
+RUN apk add --no-cache bash
+RUN apk add --no-cache attr
+
+COPY quota-crawler.sh /quota-crawler.sh
```

Comment:
Now this can be moved to csi Dockerfile and add the following in `csi/start.py`

```python
mon.add_process(Proc("quota-crawler", "/bin/bash", ["/quota-crawler.sh"]))
```